samples/java/pom_linux.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.contoso.kafka</groupId>
+    <artifactId>kafka-function</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Azure Java Functions</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <azure.functions.maven.plugin.version>1.5.0</azure.functions.maven.plugin.version>
+        <azure.functions.java.library.version>1.3.0</azure.functions.java.library.version>
+        <functionAppName>kafka-function-20190419163130420</functionAppName>
```

Comment:
Is there a way to mention that the app name need to be updated as it should be unique?

Reply:
Good point! I wrote it on the documentation, however, I'd like to find a way to write something around here.